Source/WebKit/DerivedSources.make

```diff
@@ -467,7 +467,7 @@ NOTIFICATION_ALLOW_LISTS = \
 	$(WebKit2)/Scripts/compile-sandbox.sh $@.tmp $* $(SDK_NAME) $(SANDBOX_IMPORT_DIR)
 	mv $@.tmp $@
 
-AUTOMATION_PROTOCOL_GENERATOR_SCRIPTS = \
+JSON_RPC_GENERATOR_SCRIPTS = \
```

Comment:
i feel like there should be an `AUTOMATION` or `WEBDRIVER` or something in this name so as to help future readers without context understand what it's for

Reply:
I'd rather not, as those are specific protocol names, which can be seen in context by following the target dependencies.

(It would be nice to move more of the codegen machinery out of Inspector namespace but that's a longer term project.)